### DISCUSSION TITLE
Custom SMTP issue - Recipient domain must match senders domain.

### DISCUSSION THREAD

**Discussion by akoskm** [supabase/supabase#10001] [Questions] [answered]
Users of my app reported that the signup & verification emails are arriving in the Spam folder.

I created an account on mailersend and used the SMTP credentials that the platform provided.

After applying the account details and trying to send an email with the custom SMPT settings.

It fails, and I'm getting the following error in the logs:

```
"error": "Error sending magic link email: gomail: could not send email 1: 450 Recipient domain must match senders domain. (and 1 more error) Recipient domain must match senders domain.,Recipient domain must match senders domain.",
```

**Comment by silentworks** [comment]
I have an account with Mailersend and I just tested it on of my Supabase project and it worked without any issue at all. Did you set your sender email to match that of the domain you setup on mailersend?

**Reply by akoskm** [reply]
> Did you set your sender email to match that of the domain you setup on mailersend?

I entered there an arbitrary email: noreply@tweton.com.

The sending didn't work earlier because my Mailsend account wasn't verified.

Now the email is delivered, but it's in Spam:

![image](https://user-images.githubusercontent.com/3111627/199575105-6b4bda93-b870-4cb0-8a11-b1c079f3e16f.png)

![image](https://user-images.githubusercontent.com/3111627/199575541-05b085c6-0fb1-42f8-bb16-95d4d1130c0b.png)

**Reply by silentworks** [reply]
Its probably in spam due to the content of the email. Change the email template content and let me know if its fine now.

**Reply by akoskm** [reply]
Yup, it was the contents. Thank you!

**Comment by akoskm** [comment, marked as answer]
I received the

```
"error": "Error sending magic link email: gomail: could not send email 1: 450 Recipient domain must match senders domain. (and 1 more error) Recipient domain must match senders domain.,Recipient domain must match senders domain.",
```

because despite having access to a Mailersend STMP account after completing their wizard, the Mailsend account itself wasn't verified.
